.github/workflows/publish.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish Package to npmjs
+on:
+  release:
+    types: [published]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      id-token: write
```

Comment:
Would be good to add comments why this is needed

Reply:
willfix